packages/core/src/NumberInput/NumberInput.tsx

```diff
@@ -0,0 +1,72 @@
+import { forwardRef, useMemo, useRef } from "react";
+import { useForkRef } from "@mui/material";
+import {
+  ExtractNames,
+  useDefaultProps,
+} from "@hitachivantara/uikit-react-utils";
+
+import { HvAdornment } from "../FormElement";
+import { HvIcon } from "../icons";
+import { HvInput, HvInputProps } from "../Input";
+import { changeInputValue } from "../utils/input";
+import { staticClasses, useClasses } from "./NumberInput.styles";
+
+export { staticClasses as numericInputClasses };
+
+export type HvNumberInputClasses = ExtractNames<typeof useClasses>;
```

Comment:
this is unused. we need to find a way to be able to use `HvInput`'s `classes`, and be able to override it